### PR TITLE
Update using-globus.md

### PR DESCRIPTION
**Pull request by alcf-haritha**
Added Grand globus endpoint as that is still in place and is usable. Some projects have their directories on /grand mount point.

### CODE REVIEW

File: docs/data-management/data-transfer/using-globus.md [hunk 1]
```diff
@@ -8,14 +8,15 @@ Basic documentation for getting started with Globus can be found at the followin
 [https://docs.globus.org/how-to/](https://docs.globus.org/how-to/)
 
 ## Data Transfer Node
-Several data transfer nodes (DTNs) for `/home`, Eagle, and HPSS  are available to ALCF users, allowing users to perform wide and local area data transfers. Access to the DTNs is provided via the following Globus endpoints.
+Several data transfer nodes (DTNs) for `/home`, Eagle, Grand, and HPSS  are available to ALCF users, allowing users to perform wide and local area data transfers. Access to the DTNs is provided via the following Globus endpoints.
```
Comment (felker) [inline docs/data-management/data-transfer/using-globus.md:11]:
```suggestion
Several data transfer nodes (DTNs) for `/home`, Eagle, Grand, and HPSS are available to ALCF users, allowing users to perform wide and local area data transfers. Access to the DTNs is provided via the following Globus endpoints.
```

File: docs/data-management/data-transfer/using-globus.md [hunk 2]
```diff
@@ -8,14 +8,15 @@ Basic documentation for getting started with Globus can be found at the followin
 [https://docs.globus.org/how-to/](https://docs.globus.org/how-to/)
 
 ## Data Transfer Node
-Several data transfer nodes (DTNs) for `/home`, Eagle, and HPSS  are available to ALCF users, allowing users to perform wide and local area data transfers. Access to the DTNs is provided via the following Globus endpoints.
+Several data transfer nodes (DTNs) for `/home`, Eagle, Grand, and HPSS are available to ALCF users, allowing users to perform wide and local area data transfers. Access to the DTNs is provided via the following Globus endpoints.
 
 ## ALCF Globus Endpoints
 The Globus endpoint and the path to use depends on where your data resides. If your data is on:
 
 - `/home` which is where your home directory resides: `alcf#dtn_home` for accessing `/home` (i.e. home directories on swift-home filesystem). Use the path `/<username\>`
```
Comment (felker) [inline docs/data-management/data-transfer/using-globus.md:16]:
```suggestion
- `/home` which is where your home directory resides: `alcf#dtn_home` for accessing `/home` (i.e. home directories on agile-home filesystem). Use the path `/<username\>`
```

File: docs/data-management/data-transfer/using-globus.md [hunk 3]
```diff
@@ -8,14 +8,15 @@ Basic documentation for getting started with Globus can be found at the followin
...
 - HPSS: `alcf#dtn_hpss`
 - Eagle filesystem: `alcf#dtn_eagle` for accessing /`lus/eagle/projects` or `/eagle` (i.e project directories on Eagle filesystem). Use the path `/eagle/<project name\>`
+- Grand filesystem: `alcf#dtn_grand` for accessing `/lus/grand/projects` or `/grand` (i.e. project directories on Grand filesystem). Use the path `/grand/<project name\>`
```
[the remark this answers is not in the document]
Comment (alcf-haritha) [reply docs/data-management/data-transfer/using-globus.md:19]: The fate of "Grand" is not entirely clear so I'll leave the text as-is for now.

**Review by alcf-haritha — COMMENTED** [review]
fate of grand is not fully clear, so let's leave it as-is for now.

**Pull request merged**